core/src/test/scala/unit/kafka/server/ReplicaFetcherThreadTest.scala

```diff
@@ -125,7 +125,6 @@ class ReplicaFetcherThreadTest {
     val replicaManager: ReplicaManager = mock(classOf[ReplicaManager])
     when(replicaManager.brokerTopicStats).thenReturn(mock(classOf[BrokerTopicStats]))
 
-    assertEquals(ApiKeys.FETCH.latestVersion, config.interBrokerProtocolVersion.fetchRequestVersion())
```

Comment:
Hmmm -- not sure if we want to remove this.

Reply:
If we do plan to address the MV on a followup, we should definitely call it out and file a JIRA that is a blocker for 3.7

Reply:
it looks like IBP_3_7_IV0 was added already, I was on an old version. I bumped up the fetch version so removing this isn't necessary anymore